public/app/features/plugins/admin/api.ts

```diff
@@ -35,6 +35,7 @@ export async function getPluginDetails(id: string): Promise<CatalogPluginDetails
     versions,
     statusContext: remote?.statusContext ?? '',
     iam: remote?.json?.iam,
+    changelog: remote?.changelog,
```

Comment:
Maybe we could retrieve the changelog from a local install via `http://localhost:3000/api/plugins/<plugin-id>/markdown/CHANGELOG` as we do for the readme?